src/server_manager/web_app/ui_components/app-root.ts

```diff
@@ -391,6 +395,11 @@ export class AppRoot extends polymerElementWithLocalize {
 
           <!-- Links section -->
           <paper-listbox>
+            <a class="documentation-link" 
```

Comment:
Can we hide this link if the message is not available?
Same for the other link.

Reply:
There's an option we can set that controls whether to return the message id or an empty string:
https://github.com/PolymerElements/app-localize-behavior/blob/e796bfa6876498608ab2d0d63c89bffece80c0da/app-localize-behavior.js#L274

We could also check `msg = localize('outline-documentation'); message == 'outline-documentation' || msg == ''`. Perhaps introduce a `hasMessage(id)` function?

We do need a way to gate features on the availability of messages, so the Manager release doesn't get blocked on translations. Can we make that more convenient?

In this particular case, the empty link will probably not show up, but only if localize returns empty. Showing the message id is bad.

A feature flag may be simpler and clearer than depending the existence of localization directly. That's often what we did in Web Search. Perhaps we have a `enableResourceLinks` flag, which can be hard-coded or set based on the message, and use that in the HTML?